Gemfile

```diff
@@ -1,8 +1,9 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/butteff/is_dark_ruby_gem.git" }
 
-ruby "3.2.3"
```

Comment:
"We don't need to specify the exact version of Ruby, as we have stated in the gemspec that we support versions from 2.7 and above."